src/main/java/com/bieganski/ox/App.java

```diff
@@ -2,6 +2,18 @@
 
 public class App{
 	public static void main(String[] args){
-		System.out.println("Hello world!");
+		UserInterface userInterface = new ConsoleInterface(System.in, System.out);
+		userInterface.print("Tic tac toe!");
+		Board board = new Board(15);
+		try {
+			board.addFiled(new Field(new Coordinates(2, 2), Symbol.X));
+			board.addFiled(new Field(new Coordinates(6, 7), Symbol.O));
+			board.addFiled(new Field(new Coordinates(2, 1), Symbol.X));
+			board.addFiled(new Field(new Coordinates(3, 5), Symbol.O));
+		} catch (FieldIsNotEmptyException e) {
+			e.printStackTrace();
+		}
+		userInterface.print(board);
```

Comment:
I do assume magic numbers here will be replaced later on? Or are there some particular choices behind them?

Also - how do you plan on doing automatic play happen?